dataservice/util/data_gen/data_generator.py

```diff
@@ -143,6 +145,19 @@ def _genomic_files_choices(self):
         self.max_gen_files = 5
         self.file_format_list = ['.cram', '.bam', '.vcf']
 
+    def _phenotype_choices(self):
+        """
+        Provides Choices for filling Phenotypes
+        """
+        self.min_phenotypes = 0
+        self.max_phenotypes = 8
+        pref_file = open('dataservice/util/data_gen/phenotype_hpo.csv', 'r')
+        reader = csv.reader(pref_file)
+        self.phenotype_chosen_list = []
+        for row in reader:
```

Comment:
your phenotype csv file only has one line in it. Maybe you wanted some `\n` in there?